actions/users_filter_by.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+
+'''
+Lookup user by attibutes
+'''
+
+from fnmatch import fnmatch
+from run import SlackAction
+
+
+class FilterBy(SlackAction):  # pylint: disable=too-few-public-methods
+    '''
+    Main class
+    '''
+
+    def run(self, **kwargs):
+        '''
+        Return a list of users
+        '''
+
+        users = []
+        attrs = kwargs.pop('attributes')
+        res = super(FilterBy, self).run(**kwargs)
+
+        for user in res['members']:
+            match = True
+            for key, val in attrs.items():
```

Comment:
I'd rather this be `iteritems`, but I don't consider it a blocker for this PR since I can't imagine the parameters will be large enough to matter.

Reply:
Yep, just until packs get updated to py3 then this practice could have some negative memory usage side effects. Would be better to use `iteritems` from `six` package to make it work for 2/3. Certainly not a blocker in this usage, but something to keep an eye on in the future.